ucf-events.php

```diff
@@ -2,7 +2,7 @@
 /*
 Plugin Name: UCF Events
 Description: Contains shortcode and widget for displaying UCF Events Feeds
-Version: 2.1.3
+Version: 2.1.4
 Author: UCF Web Communications
```

Comment:
For the sake of being thorough in our documentation, we should probably add a minimum php version tag to the plugin meta to ensure it is documented. The following line can be added underneath this version to document the minimum version requirement: `Requires PHP: 5.4`.